view/script/patient.js

```diff
@@ -38,7 +39,9 @@
         }
     };
 
+    var allDatesConfig = jQuery.extend(true, {}, window.dates);
```

Comment:
I wanted to use copy of object (deep) to make some changes so that window.dates should not get affected. Is there a better way of doing it?